daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/command/Command.scala

```diff
@@ -73,8 +93,8 @@ final case class CreateAndExerciseCommand(
   *    interpretation will take this instant
   *  @param commandsReference id passed only for error reporting
   */
-case class Commands(
-    commands: ImmArray[Command],
+final case class Commands(
+    commands: ImmArray[ApiCommand],
```

Comment:
@gerolf-da be aware I change the API type here.

Reply:
Thanks for the ping. LGTM.